task_DEV_3/task_DEV_3/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+
+namespace task_DEV_3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter the size of the array :");
```

Comment:
If you write in message `enter size` then it is better to name variable like `size`. Or if you want to name variable like `length `then in message you need to write `length`. Just to avoid misunderstanding.

Reply:
done